modular_skyrat/modules/food_replicator/reagents.dm

```diff
@@ -0,0 +1,96 @@
+/datum/reagent/consumable/powdered_tea
+	name = "Powdered Tea"
+	description = "Tea in its powdered form. Tastes horribly."
+	color = "#3a3a03"
+	nutriment_factor = 0
+	taste_description = "bitter powder"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+	default_container = /obj/item/reagent_containers/cup/glass/mug/tea
+
+/datum/chemical_reaction/food/unpowdered_tea
+	required_reagents = list(
+		/datum/reagent/water = 1,
+		/datum/reagent/consumable/powdered_tea = 1,
+	)
+	results = list(/datum/reagent/consumable/tea = 2)
+	mix_message = "The mixture instantly heats up."
+	reaction_flags = REACTION_INSTANT
+
+/datum/reagent/consumable/powdered_coffee
+	name = "Powdered Coffee"
+	description = "Americano in its powdered form. Quite an ordinary thing to be honest."
+	color = "#101000"
+	nutriment_factor = 0
+	taste_description = "very bitter powder"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+	default_container = /obj/item/reagent_containers/cup/glass/coffee
+
+/datum/chemical_reaction/food/unpowdered_coffee
+	required_reagents = list(
+		/datum/reagent/water = 1,
+		/datum/reagent/consumable/powdered_coffee = 1,
+	)
+	results = list(/datum/reagent/consumable/coffee = 2)
+	mix_message = "The mixture instantly heats up."
+	reaction_flags = REACTION_INSTANT
+
+/datum/reagent/consumable/powdered_coco
+	name = "Powdered Coco"
+	description = "Made with love (citation needed), and reclaimed biomass."
+	nutriment_factor = 0
+	color = "#403010"
+	taste_description = "dry chocolate"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+	default_container = /obj/item/reagent_containers/cup/glass/mug/coco
+
+/datum/chemical_reaction/food/unpowdered_coco
+	required_reagents = list(
+		/datum/reagent/water = 1,
```

Comment:
Make it use milk, since it's what the regular hot coco recipe asks for. It'd be a bit weird to obtain something creamy out of just water, no? Might also want to make it require heating as well, since otherwise it's not really hot coco, it'd be chocolate milk, really.
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/58045821/8029fa32-5a29-4de5-b0a4-ebcf5fb092d3)

```suggestion
		/datum/reagent/milk = 1,
```

Reply:
Will do, makes sense.